diophantine.tex

```diff
@@ -49,5 +50,9 @@
 42*t1**2 + 42*t2**2 + 42*t3**2 + 42*t4**2 + 42*t5**2)])
 
 >>> diophantine(a**2 + b**2 + c**2 + d**2 + e**2 + f**2 - 112)
-set([(8, 4, 4, 4, 0, 0)])
+set([(2, 3, 3, 4, 5, 7), (2, 2, 4, 4, 6, 6), (0, 3, 3, 3, 6, 7), (0, 2, 3, 5, 5,
```

Comment:
Oh, I see. Updates to 1.0 are not included. My edits to the diophantine doc will have to change a bit, too.